src/compiler/symbols/Signature.ts

```diff
@@ -66,4 +66,12 @@ export class Signature {
         const tags = this.compilerSignature.getJsDocTags();
         return tags.map(t => this._context.compilerFactory.getJSDocTagInfo(t));
     }
+
+    /**
+     * Gets the signature's declaration.
+     */
+    getDeclaration(): SignaturedDeclaration {
```

Comment:
The explicit type can be removed here in order to leave this up to the typescript compiler based on the type of `compilerSignatureDeclaration`. For example:

```ts
    getDeclaration() {
        const { compilerFactory } = this._context;
        const compilerSignatureDeclaration = this.compilerSignature.getDeclaration();
        return compilerFactory.getNodeFromCompilerNode(compilerSignatureDeclaration, compilerFactory.getSourceFileForNode(compilerSignatureDeclaration));
    }
```

Reply:
fixed this in #549